VAMobile/src/constants/common.ts

```diff
@@ -48,4 +48,12 @@ export const MAX_DIGITS_AFTER_FORMAT = 14
 export const EMAIL_REGEX_EXP = new RegExp(
   /^(([^<>()\]\\.,;:\s@"]+(\.[^<>()\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
 )
+export const MAIL_TO_REGEX_EXP = new RegExp(
```

Comment:
it is the same regex with mailto: on it. It however was difficult for me to concat it as the email regex is a RegEx type instead of string. I elected to keep them separate to also help differentiate which ones should be applied to each of the checks.